### ISSUE TITLE
Unsupported import file extension "yaml" Import

### ISSUE THREAD

**Issue by rohrbachger** [romainsi/zabbix-VB-R-SQL#1]
With the 5.0 Version it seems not to work.
As I can´t upgrade easy, is there a template available for this version?

I have running an very old version with powershell scripts, and having timeout issues with the large file.
I would like to use the DB version, this makes much more sense I guess.

**Comment by romainsi** [comment]
Hello,

I made an export in .xml but I think you're going to get errors on import. It should be possible to modify for adapt it (especially the trigger times in minutes that were in seconds on older versions and actions that may not be present to remove).

**Comment by rohrbachger** [comment]
I tried this, yes. And run into the issues. 
It takes time to understand what is to change, I hoped there is an easier way.
One problem was easy. I found a converter yaml to xml.
You used for name "Backup &  Recovery". The & is bad, but with search and replace was easy solved.
The next issue needs more understanding, so I did not looked at this.

I like the sql version, I remember several years ago I did for BackupExec,Veriats the same.

**Comment by rohrbachger** [comment]
I tried with your xml template again, I only modified the export version to be able to read.
No luck, still
Invalid tag "/zabbix_export/templates/template(1)/items/item(2)/preprocessing/step(1)": unexpected tag "parameters".
The structure seems different,

**Comment by techinnov-974** [comment]
Hello @rohrbachger  Thank you for reporting the problem yesterday I was in the same case, thanks to your help I managed to debug the code and it is imported correctly from my side.

@romainsi  The xml file does not import in version 5.0.8 of zabbix, I had to correct the tags because I have the same error as @rohrbachger 

Problem come from the tags in `"<preprocessing>"` the tags `"<parameters>"` and `"<parameter>"` caused concern.

**However, I have not tested the operation. Check if it works**

Download XML : [Template Veeam Backup And Replication new.zip](https://github.com/romainsi/zabbix-VB-R-SQL/files/5969721/Template.Veeam.Backup.And.Replication.new.zip)

![image](https://user-images.githubusercontent.com/29580589/107731013-4a647780-6d0e-11eb-9291-33c23cdad104.png)
